static/templates/user_info_popover_content.hbs

```diff
@@ -1,79 +1,82 @@
-        {{#each display_profile_fields}}
-            <li data-type="{{this.type}}" class="field-section custom_user_field tippy-zulip-tooltip" data-tippy-content="{{this.name}}" data-field-id="{{this.id}}">
-                {{#if this.is_link}}
-                    <a tabindex="0" href="{{this.value}}" target="_blank" rel="noopener noreferrer" class="value">{{this.value}}</a>
-                {{else if this.is_external_account}}
-                    <a tabindex="0" href="{{this.link}}" target="_blank" rel="noopener noreferrer" class="value">
-                        {{#if (eq this.subtype "github") }}
-                        <i class="fa fa-github" aria-hidden="true"></i>
-                        {{else if (eq this.subtype "twitter") }}
-                        <i class="fa fa-twitter" aria-hidden="true"></i>
-                        {{/if}}
-                        {{this.value}}
-                    </a>
-                {{else}}
-                    {{#if this.rendered_value}}
-                    <span class="value rendered_markdown">{{rendered_markdown this.rendered_value}}</span>
-                    {{else}}
-                    <span class="value">{{this.value}}</span>
+    {{else}}
+        <li>{{ user_type }}</li>
+    {{/if}}
+    {{!-- Display selected custom profile fields in this popover. --}}
+    {{#each display_profile_fields}}
+        <li data-type="{{this.type}}" class="field-section custom_user_field tippy-zulip-tooltip" data-tippy-content="{{this.name}}" data-field-id="{{this.id}}">
+            {{#if this.is_link}}
+                <a tabindex="0" href="{{this.value}}" target="_blank" rel="noopener noreferrer" class="value custom_profile_fields_link">{{this.value}}</a>
+            {{else if this.is_external_account}}
+                <a tabindex="0" href="{{this.link}}" target="_blank" rel="noopener noreferrer" class="value custom_profile_fields_link">
+                    {{#if (eq this.subtype "github") }}
+                    <i class="fa fa-github" aria-hidden="true"></i>
+                    {{else if (eq this.subtype "twitter") }}
+                    <i class="fa fa-twitter" aria-hidden="true"></i>
                     {{/if}}
+                    {{this.value}}
+                </a>
+            {{else}}
+                {{#if this.rendered_value}}
+                <span class="value rendered_markdown">{{rendered_markdown this.rendered_value}}</span>
+                {{else}}
+                <span class="value">{{this.value}}</span>
                 {{/if}}
-            </li>
-        {{/each}}
-
-        {{#if user_time}}
-            <li class="hidden-for-spectators">{{ user_time }} {{#tr}}Local time{{/tr}}</li>
-        {{/if}}
+            {{/if}}
+        </li>
+    {{/each}}
 
-        <li class="only-visible-for-spectators">Joined {{date_joined}}</li>
-    </div>
+    {{#if (and user_time is_active)}}
+        <li class="hidden-for-spectators" >{{ user_time }} {{#tr}}local time{{/tr}}</li>
```

Comment:
I'm pretty sure the user_time variable should be inside the translation box; this is required for languages with a different word order. Fixing this will also remove the need to add anything to `capitalization.py`.

Se https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.

Probably best to fix both of these bugs in a separate commit, since they're preexisting.